### ISSUE TITLE
Where is supplement mentioned in the paper

### ISSUE THREAD

**Issue by steven5401** [mseg-dataset/mseg-api#19]
Hello, I want to know the datasets that were not used in MSeg and the reasons for not including them. However, I can not find supplement through google. Can you give me the link of supplement?

**Comment by johnwlambert** [comment]
Hi @steven5401 , thanks for your interest in our work. The supplement will be included in an upcoming expanded Arxiv preprint. Here's a table that should address your question:

<img width="982" alt="Screen Shot 2020-09-01 at 10 07 37 PM" src="https://user-images.githubusercontent.com/16724970/91923952-94492c00-ec9f-11ea-9be1-e23c6688183e.png">

Let us know if you have further questions.

**Comment by steven5401** [comment]
Thank you very much. Why didn't you choose scannet as training set, but test set?

**Comment by johnwlambert** [comment]
No problem. We had a large number of concerns about the quality of the 2d labels in ScanNet. As you know, ScanNet is also annotated in 3d, not 2d. Unlike Matterport3d and Stanford-2d-3d, ScanNet does have a 2d label benchmark. Because of its noisy nature, we didn't want to train on it, but it is a good proxy for performance in the real-world.

Here are a few examples from ScanNet:

Book vs. bookshelf seems arbitrary:
<img width="758" alt="Screen Shot 2020-09-02 at 11 18 57 AM" src="https://user-images.githubusercontent.com/16724970/92003071-939bae80-ed0e-11ea-806b-ec0b6f2eb196.png">
The non-white pixels in the center should be "Wall", but include "blinds"
<img width="762" alt="Screen Shot 2020-09-02 at 11 18 30 AM" src="https://user-images.githubusercontent.com/16724970/92003072-939bae80-ed0e-11ea-8694-1d6582e8f39c.png">
The door label extends onto the glass window:
<img width="767" alt="Screen Shot 2020-09-02 at 11 18 09 AM" src="https://user-images.githubusercontent.com/16724970/92003073-94344500-ed0e-11ea-835e-2ac9f3c51fe2.png">

Bed and backpack boundaries are poorly aligned:
<img width="761" alt="Screen Shot 2020-09-02 at 11 16 43 AM" src="https://user-images.githubusercontent.com/16724970/92003076-94344500-ed0e-11ea-96ec-8a33f19a0ca6.png">
Bicycle extent is quite incorrect:
<img width="759" alt="Screen Shot 2020-09-02 at 11 16 26 AM" src="https://user-images.githubusercontent.com/16724970/92003077-94ccdb80-ed0e-11ea-8855-0e55e262887d.png">

**Comment by steven5401** [comment]
I understand it. Thank you for the prompt reply.
